docs/source/getting-started/first-feature.rst

```diff
@@ -68,16 +68,11 @@ This completes the workflow for messaging in our app.
 Running the New Feature
 =======================
 
-We need to terminate the previous ``daml start`` process and run it again, as we need to have a Sandbox instance with a DAR file containing the new feature. As a reminder, by running ``daml start`` again we will
+Navigate to the terminal window where the ``daml start`` process is running and press 'r'. This will
```

Comment:
Hmm, we might want to suggest pressing Enter/Return on Windows. Not sure about the actual wording though.

```suggestion
Navigate to the terminal window where the ``daml start`` process is running and press 'r'. On Windows, you must also press the return key. This will
```